packages/ndla-ui/src/RadioButtonGroup/component.radio-button-group.scss

```
@@ -42,6 +42,7 @@
   &__input {
     opacity: 0;
     position: absolute;
+    display:none;
```

```suggestion
    display: none;
```

var det ingen måte å fjerne hele den diven/ boksen? Det du gjør nå er jo bare å gjøre den ""usynlig"", ikke faktisk fjerne den.

---

Det er helt sant, men saken er at den/de divene er faktisk selve RadioButtons so that makes it complicated, og måten jeg kom rundt det, kun for å fikse det rare problemet er å gjøre det usynlig. Weird enough så gjør `display: none` ikke selve RadioButtons usynlig men den rare tynne diven. Jeg skjønner ikke åssen det fungerer, but it works.